Circuit/Components/Triode.cs

```diff
@@ -97,6 +126,13 @@ public override void Analyze(Analysis Mna)
                     ip = (Call.Max(E1, 0) ^ Ex) / Kg;
                     ig = Call.Max(Vgk - Vg, 0) / Rgk;
                     break;
+                case TriodeModel.DempwolfZolzer:
+                    var cgVgk = Cg * Vgk;
+                    ig = Call.If(Vgk > -5, Gg * Binary.Power(Call.If(cgVgk < 5, Call.Ln(1 + LinExp(cgVgk)), cgVgk) / Cg, Xi), 0) + Ig0;
+                    Expression ex = C * ((Vpk / Mu) + Vgk);
+                    var ik = Call.If(ex > -5, G * Binary.Power(Call.If(ex < 5, Call.Ln(1 + LinExp(ex)), ex) / C, Gamma), 0);
```

Comment:
Yeah, basically this pull request is a quick copy and paste from my "playground" branch which is based on your 2015's code base. I've seen some movement in this repository recently and I thought that some of my changes might be useful. I'll refactor this part a little bit with Ln1Exp(x). If I remember correctly adding the limit at the low end was caused by some stability problems, or maybe i was looking for a little bit more performance (but I don't think it improved something because of how conditional statements are generated by the CodeGen using intermediate variables, which causes  both branches to be calculated before the conditional statement). There's also a room to refactor here, as models and UI related stuff could be separated from the component classes.

Reply:
Okay, I've tried without low end limit and the simulation has become unstable.